jetty-documentation/src/main/asciidoc/administration/startup/startup-modules.adoc

```diff
@@ -95,14 +95,109 @@ include::screen-http-webapp-deploy-listconfig.adoc[]
+
+As an alternative to a single `start.ini` file you can opt to house modules in a `${jetty.base}/start.d` directory.
+Modules activated when a `start.d` directory exists will be stored as a single file per module.
+Below is an example of a fresh `${jetty.base}` that will create a `start.d` directory and activate several modules.
+
+[source, screen, subs="{sub-order}"]
+....
+[jetty.home]$ mkdir mybase
+[jetty.home]$ cd mybase/
+[mybase]$ java -jar ../start.jar --create-startd
+INFO : Base directory was modified
+[mybase]$ ls -all
+total 0
+drwxr-xr-x   3 staff  staff  102 Aug 29 15:16 .
+drwxr-xr-x@ 26 staff  staff  884 Aug 29 15:16 ..
+drwxr-xr-x   6 staff  staff  204 Aug 29 15:19 start.d
+
+[mybase]$ java -jar ../start.jar --add-to-start=server,client,webapp,websocket
+INFO : webapp          initialised in ${jetty.base}/start.d/webapp.ini
+INFO : server          initialised in ${jetty.base}/start.d/server.ini
+INFO : websocket       initialised in ${jetty.base}/start.d/websocket.ini
+INFO : client          initialised in ${jetty.base}/start.d/client.ini
+INFO : Base directory was modified
+[mybase]$ cd start.d/
+[mybase]$ ls -all
+total 32
+drwxr-xr-x  6 staff  staff   204 Aug 29 15:19 .
+drwxr-xr-x  3 staff  staff   102 Aug 29 15:16 ..
+-rw-r--r--  1 staff  staff   175 Aug 29 15:19 client.ini
+-rw-r--r--  1 staff  staff  2250 Aug 29 15:19 server.ini
+-rw-r--r--  1 staff  staff   265 Aug 29 15:19 webapp.ini
+-rw-r--r--  1 staff  staff   177 Aug 29 15:19 websocket.ini
+....
+
+In the example, we first create a new `${jetty.base}` and then create the `start.d` directory with the `--create-startd` command.
+Next, we use the `--add-to-start` command which activates the modules and creates their respective ini files in the `start.d` directory.
+
+If you have an existing `start.ini` file but would like to use the `start.d` structure for additional modules, you can use the `--create-startd` command as well.
+Doing this will create the `start.d` directory and copy your existing `start.ini` file in to it.
+Any new modules added to the server will have their own `<module name>.ini` file created in the `start.d` directory.
+
+[source, screen, subs="{sub-order}"]
+....
+[mybase]$ java -jar ../start.jar --add-to-start=server,client,webapp,websocket
+INFO : webapp          initialised in ${jetty.base}/start.ini
+INFO : server          initialised in ${jetty.base}/start.ini
+INFO : websocket       initialised in ${jetty.base}/start.ini
+INFO : client          initialised in ${jetty.base}/start.ini
+INFO : Base directory was modified
+[mybase]$ ls -all
+total 8
+drwxr-xr-x   3 staff  staff   102 Aug 29 15:24 .
+drwxr-xr-x@ 26 staff  staff   884 Aug 29 15:16 ..
+-rw-r--r--   1 staff  staff  2867 Aug 29 15:24 start.ini
+
+[mybase]$ java -jar ../start.jar --create-startd
+INFO : Base directory was modified
+[mybase]$ ls -all
```

Comment:
maybe use the tree command so you can see a whole hierarchy rather that cd xxx; ls; cd .. 
or at least just do ls xxx ?